cfy_manager/components/stage/stage.py

```diff
@@ -115,7 +115,8 @@ def _install(self):
 
     def _create_user_and_set_permissions(self):
         create_service_user(STAGE_USER, STAGE_GROUP, HOME_DIR)
-
+        # stage user is in the cfyuser group for replication
+        common.sudo(['usermod', '-aG', CLOUDIFY_GROUP, STAGE_USER])
```

Comment:
just moving it here, near where the user is created, instead of where it was - in some unrelated function